components/datetime/src/format/datetime.rs

```diff
@@ -712,174 +714,160 @@ where
 
 /// Given a [`ResolvedNeoTimeZoneSkeleton`], select the formatter units
 fn select_zone_units(time_zone: ResolvedNeoTimeZoneSkeleton) -> [Option<TimeZoneFormatterUnit>; 3] {
-    // Select which formatters to try based on the field.
-    let mut formatters = (
-        None,
-        None,
-        // Friendly Localized offset Format (requires "essentials" data)
-        Some(TimeZoneFormatterUnit::WithFallback(
-            FallbackTimeZoneFormatterUnit::LocalizedOffset(LocalizedOffsetFormat {}),
-        )),
-    );
     match time_zone {
         // `z..zzz`
-        ResolvedNeoTimeZoneSkeleton::SpecificShort => {
-            formatters.0 = Some(TimeZoneFormatterUnit::SpecificNonLocationShort(
-                SpecificNonLocationShortFormat {},
-            ));
-        }
+        ResolvedNeoTimeZoneSkeleton::SpecificShort => [
+            Some(TimeZoneFormatterUnit::SpecificNonLocationShort),
+            Some(TimeZoneFormatterUnit::LocalizedOffsetLong),
```

Comment:
UTS 35 says that Specific Short should go to Short Localized GMT

https://unicode.org/reports/tr35/tr35-dates.html#goals

Reply:
Done